src/main/kotlin/com/terraformation/backend/documentproducer/DocumentUpgradeCalculator.kt

```diff
@@ -0,0 +1,266 @@
+package com.terraformation.pdd.document
```

Comment:
Were the package names supposed to be updated per the PR description?

Reply:
Sorry, I will update the description, the files were moved into the new folder structure, but as mentioned at the top of the description, none of the contents have changed. All of the modifications are made in a subsequent PR.